konsave/consts.py

```diff
@@ -18,6 +18,3 @@
 
 list_of_profiles = os.listdir(PROFILES_DIR)
 length_of_lop = len(list_of_profiles)
```

Comment:
Renamed the `vars.py` module to `consts.py`, since it contains package-level constants.